Chat/Using chat to send messages and make voice calls/index.js

```diff
@@ -0,0 +1,99 @@
+require("dotenv").config()
+
+const auth = {
+    username: process.env.PROJECT_ID,
+    password: process.env.API_TOKEN
+}
+
+const apiUrl = `https://${process.env.SPACE_URL}`
+
+const axios = require("axios")
+const express = require("express")
+const bodyParser = require("body-parser")
+const cors = require("cors")
+const { Voice, Messaging } = require("@signalwire/realtime-api")
+
+const app = express()
+const port = 3000
+
+app.use(bodyParser.json())
+app.use(cors())
+app.use(express.static('frontend'))
+
+let client = new Voice.Client({
+    project: process.env.PROJECT_ID,
+    token: process.env.API_TOKEN,
+    contexts: ["user"]
+});
+
+let messageClient = new Messaging.Client({
+    project: process.env.PROJECT_ID,
+    token: process.env.API_TOKEN,
+    contexts: ['user']
+})
+
+app.post("/get_chat_token", async function (req, res) {
+
+    const { member_id, channels } = req.body
+
+    const channelsPerms = {}
+
```

Comment:
If we're limiting this IVR to one channel, do you want to hard code the single channel for the chat token?

Reply:
The channel is value is passed whenever the user is trying to create a chat room. i am changing the parameter name to channel instead of channels